blossom/templates/website/generic_form.html

```diff
@@ -1,14 +1,70 @@
 {% extends 'website/partials/base.partial' %}
 {% load widget_tweaks %}
 {% load static %}
+{% load string_helpers %}
 
 {% block content %}
-    <h1>{{ header }}</h1>
-    <h4>{{ subheader }}</h4>
+    {% comment %}
+    TODO: Rip jQuery out of this form
+    This has just enough jquery in it to be irritating, but it's the only
+    page that actually requires it. Since this is the only one, we'll
+    import jquery here and hopefully nuke it one of these days when someone
+    better at JS than me comes and looks at it.
```

Comment:
Let's do this now, if we merge this we'll probably forget about this and have it forever as an additional dependency. I took an initial look at the code and I think this is not too hard to accomplish with normal JS.

Reply:
Is this something that you can sort? I honestly don't care enough about it to warrant pulling out jquery right now, especially because this particular page is only loaded by staff.

Reply:
Yes I might have time to do it today